com.unity.render-pipelines.core/Runtime/Lighting/ProbeVolume/ProbeBrickPool.cs

```diff
@@ -7,6 +7,8 @@ namespace UnityEngine.Experimental.Rendering
 {
     internal class ProbeBrickPool
     {
+        const int kProbeIndexPoolAllocationSize = 128;
```

Comment:
The word Index here is _very_ confusing, this is just the Pool. 

I'd just call this kProbePoolChunkSize

Reply:
(I know was there before, but it was bad :P , might as well change it )

Reply:
I agree with the change. Will do.